File: rcp10/B.py
aux = input()
a = int( aux[0:4])
b = int(aux[5:9])
c = aux[10:11]
ans = 0 
for i in range(a,b+1):
    x = str(i)
    for j in x:
        if(j == c):
            ans+=1
print(ans)

File: rcp10/B.cpp
#include <bits/stdc++.h>
#include <stdlib.h>
#include <stdio.h>
 
using namespace std; 
int digitos(int i,int c)
{
    int ans = 0;
    string aux;
    itoa(i,aux,10);
    
}
int main()
{
    ios::sync_with_stdio(0); cin.tie(0);
    int a,b,c;
    long long ans;
    cin>>a>>b>>ans;
    ans=0;
    for(int i=a;i <=b;i++)
    {
        ans+=digitos(i,c);
    }
}